analysis/tests/parameter_tuning_test.py

```diff
@@ -245,15 +249,15 @@ def test_tune_count(self):
         # Assert.
         tune_result, per_partition_utility_analysis = result
         per_partition_utility_analysis = list(per_partition_utility_analysis)
-        self.assertLen(per_partition_utility_analysis, 40)
```

Comment:
so test was incorrect in the past and 40 and 4 below are incorrect expectations?

Reply:
`tune()` incorrectly worked, because `max_value` was incorrect.